Packs/AlphaVantage/pack_metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "AlphaVantage",
+    "description": "API for stocks prices data",
```

Comment:
-> The Alpha Vantage content pack provides accessible APIs for financial market data such as stock prices. Utilize this pack to get open stock prices, high/low price, trade volume, and so on.